src/shared/utils/getDeclarationErrorAddendum.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule getDeclarationErrorAddendum
```

Comment:
I've been mostly moving away from the "Check the render method of" in favor of just showing the component stack.